.github/filters.yml

```diff
@@ -1,31 +1,48 @@
+# Also see list of stacks in the job `add-stack-label` under workflow
```

Comment:
Is there a way to check that _all_ files are covered in `filters.yml`? What if we add some top-level folder in the future, and forget to add it here?

Reply:
I don't know if there is a pre-existing way to check that. It's likely that we'll have to build something like that ourselves.